library.json

```
@@ -10,7 +10,7 @@
     "type": "git",
     "url": "https://github.com/esphome/AsyncTCP.git"
   },
-  "version": "2.1.3",
+  "version": "2.1.4",
```

Library version should not be bumped in PRs like this
```suggestion
  "version": "2.1.3",
```